src/components/widget-creator.jsx

```diff
@@ -384,6 +384,18 @@ const WidgetCreator = ({instId, widgetId, minHeight='', minWidth=''}) => {
 	}
 
 	const save = (instanceName, qset, version = 1) => {
+        //cancel saving of the widget if title is too long to prevent crashing
+        let titleLength = instanceName.length;
+        if(titleLength>100) {
```

Comment:
`titleLength` definition is unnecessary, you can reference `instanceName.length` here directly